docs/terminal/brisbane.md

```diff
@@ -109,66 +108,69 @@ All other aircraft must be voice coordinated to BN TCU.
 Auto Release shall be used for aircraft that are:    
 a) Departing from a runway nominated on the ATIS; and  
 b) Issued a Procedural SID; and   
-c) Assigned the standard assignable level.
+c) Assigned the standard assignable level.  
 
-Any aircraft that don't meet these criteria must be coordinated to BN TCU with a "Next" Call.
+Any aircraft that don't meet these criteria must be coordinated to BN TCU with a "Next" Call.  
 
 "Next" Coordination is a procedure where the BN ADC controller gives a heads-up to the BN TCU controller about an impending departure. The BN TCU controller will respond by assigning a heading to the aircraft, for the BN ADC controller to pass on with their takeoff clearance.
 
 !!! example
     **BN ADC** -> **BN TCU**: "Next, ABC"  
     **BN TCU** -> **BN ADC**: "ABC, Heading 030"  
     **BN ADC** -> **BN TCU**: "Heading 030, ABC"  
-    **BN ADC** -> **ABC**: "ABC, Assigned heading right 030, Runway 34R, Cleared for Takeoff"  
-    **ABC** -> **BN ADC**: "Right heading 030, Runway 34R, Cleared for Takeoff, ABC"  
+
+    **BN ADC**: "ABC, Assigned heading right 030, Runway 34R, Cleared for Takeoff"  
+    **ABC**: "Right heading 030, Runway 34R, Cleared for Takeoff, ABC"  
     `AIP GEN 3.4`
 
-### BN TCU / AF TWR
+### BN TCU / AF ADC
 #### Departures
 
-Aircraft departing YBAF in to BN TCU Class C will be coordinated from **AF TWR** at Taxi. Aircraft will need to be passed airways clearances to **AF TWR** at this point, to be relayed to the aircraft.
+Aircraft departing YBAF in to BN TCU Class C will be coordinated from **AF ADC** at Taxi. Aircraft will need to be passed airways clearances to **AF ADC** at this point, to be relayed to the aircraft.
 
 The Standard assignable levels for YBAF departures is the lower of `A040` or the `RFL`.
 
 !!! example
-    **AF TWR** -> **BN TCU**: "Taxi, DEF, VFR for YCDR via TVT"  
-    **BN TCU** -> **AF TWR**: "DEF, AF, TVT, Flight Planned Route, A040, Squawk 3601"  
-    **AF TWR** -> **BN TCU**: "AF, TVT, Flight Planned Route, A040, Squawk 3601, DEF"  
-    **AF TWR** Will then pass the airways clearance to DEF
+    **AF ADC** -> **BN TCU**: "Taxi, DEF, VFR for YCDR via TVT"  
+    **BN TCU** -> **AF ADC**: "DEF, AF, TVT, Flight Planned Route, A040, Squawk 3601"  
+    **AF ADC** -> **BN TCU**: "AF, TVT, Flight Planned Route, A040, Squawk 3601, DEF"  
+
+    **AF ADC** Will then pass the airways clearance to DEF  
+    `AIP GEN 3.4`
 
 #### Arrivals
 
-YBAF arrivals shall be coordinated to **AF TWR** from the relevant BN TCU controller prior to transfer of jurisdiction.
+YBAF arrivals shall be coordinated to **AF ADC** from the relevant BN TCU controller prior to transfer of jurisdiction.
 
 !!! example
-    **BN TCU** -> **AF TWR**: "BDF, via HUUGO, Number 1"  
-    **AF TWR** -> **BN TCU**: "BDF, Number 1"  
+    **BN TCU** -> **AF ADC**: "BDF, via HUUGO, Number 1"  
+    **AF ADC** -> **BN TCU**: "BDF, Number 1"  
```

Comment:
In reality Metro D's don't receive a sequence number. BDF, via HUUGO <at time> should suffice (without the time)